packages/playground-examples/copy/ja/4-0/New JS Features/JSDoc Deprecated.ts

```diff
@@ -0,0 +1,19 @@
+// 4.0では、JSDocのタグである@deprecated(非推奨)が型システムに
```

Comment:
IMO: JSDocのものは訳さなくでもいいかなと思います

```suggestion
// 4.0では、JSDocのタグである@deprecatedが型システムに
```